File: Ruby/season2/p015-018/p015.rb
# -*- coding: utf-8 -*-
=begin
　2017.5.2
　テキストファイル読み込み
=end

def textread (fn)
  # ファイル指定子の作成
  fp = open(fn)

  # 文字列読み出し
  s = fp.read

  # 表示
  print s

  # ファイルを閉じる
  fp.close
end


def main
  # 指定されたファイルを開いて表示
  textread ARGV[0]
end


main

File: Python/season2/p015-016/p015.py
# coding: UTF-8
"""
　2018.4.13
　条件分岐の整理
"""

def check_num_size(x):
    if 2 <= x < 3:
        print ("2 以上 3 未満")
    elif 1 <= x < 2:
        print ("1 以上 2 未満")
    elif x < 1:
        print ("1 未満")
    else:
        print("3 以上")

check_num_size(-1)
check_num_size(10)